src/features/device-state/routes/state-get-v2.ts

```diff
@@ -188,6 +183,21 @@ const stateQuery = _.once(() =>
 											is_a_build_of__service: {
 												$select: ['id', 'service_name'],
 												$expand: {
+													device_service_environment_variable: {
+														$select: ['name', 'value'],
+														$filter: {
+															device: {
```

Comment:
We need this $filter (or some variation of it) as otherwise the expand will get all `device_service_environment_variable` for that server and ignore the `device` filter

Reply:
Some other idea would be to directly expand from the device on the top level to `dsev` but then we need to do something similar for service (which isa bit longer)